src/domains/sign/io_sign_support_func.tf

```diff
@@ -0,0 +1,179 @@
+locals {
+  io_sign_support_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
+      AzureWebJobsDisableHomepage    = "true"
+      NODE_ENV                       = "production"
+      CosmosDbConnectionString       = module.cosmosdb_account.connection_strings[0]
+      CosmosDbIssuerDatabaseName     = module.cosmosdb_sql_database_issuer.name
+      CosmosDbUserDatabaseName       = module.cosmosdb_sql_database_user.name
+      PdvTokenizerApiBasePath        = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerApiKey             = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+    }
+  }
+}
+
+module "io_sign_support_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v6.1.0"
+
+  name                = format("%s-support-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  health_check_path = "/api/v1/sign/support/info"
+
+  always_on = true
+
+  runtime_version = "~4"
+  node_version    = "18"
+
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = var.io_sign_support_func.sku_tier
+    sku_size                     = var.io_sign_support_func.sku_size
+    maximum_elastic_worker_count = 0
+  }
+
+  app_settings = local.io_sign_support_func.app_settings
+
+  subnet_id       = module.io_sign_support_snet.id
+  allowed_subnets = [module.io_sign_support_snet.id, data.azurerm_subnet.apim.id]
```

Comment:
Since this function does not need to be called by the APIM, I would remove `data.azurerm_subnet.apim.id`